src/validation.h

```diff
@@ -523,6 +521,14 @@ class CChainState {
      */
     CCriticalSection m_cs_chainstate;
 
+    /**
+     * Is this chainstate undergoing initial block download?
```

Comment:
in commit 61fd34e244  refactoring: IsInitialBlockDownload -> CChainState


Instead of a question, say "Whether this c..."?